modules/luci-base/luasrc/view/wifi_assoclist.htm

```diff
@@ -68,13 +71,21 @@
 	);
 //]]></script>
 
+<script type="text/javascript">//<![CDATA[
+	function kick_client(iface, mac) {
```

Comment:
Please add the function to the existing `<script>` block, rename it to `function handleDisconnect(ev) {` and obtain iface and bssid using:

```
var iface = ev.target.getAttribute('data-iface'),
    bssid = ev.target.getAttribute('data-bssid');
```